roles/redpanda_broker/defaults/main.yml

```diff
@@ -54,12 +54,3 @@ rp_repo_signing_deb: "deb [signed-by=/usr/share/keyrings/redpanda-redpanda-archi
 rp_repo_signing_deb_unstable: "deb [signed-by=/usr/share/keyrings/redpanda-redpanda-unstable-archive-keyring.gpg] {{ redpanda_base_url }}/E4xN1tVe3Xy60GTx/redpanda-unstable/deb/{{ansible_distribution | lower}} {{ ansible_distribution_release | lower }} main"
 rp_repo_signing_src_deb: "deb-src [signed-by=/usr/share/keyrings/redpanda-redpanda-archive-keyring.gpg] {{ redpanda_base_url }}/public/redpanda/deb/{{ansible_distribution | lower}} {{ ansible_distribution_release | lower }} main"
 rp_repo_signing_src_deb_unstable: "deb-src [signed-by=/usr/share/keyrings/redpanda-redpanda-unstable-archive-keyring.gpg] {{ redpanda_base_url }}/E4xN1tVe3Xy60GTx/redpanda-unstable/deb/{{ansible_distribution | lower}} {{ ansible_distribution_release | lower }} main"
-debian_prerequisite_packages:
-  - debian-keyring
-  - debian-archive-keyring
-  - apt-transport-https
-  - ca-certificates
-  - gnupg
-  - iotop
-  - mdadm
-  - xfsprogs
```

Comment:
these two are still needed for when the user wants to use our separate playbook for setting up /var/lib/redpanda on linux raid, right? Did this get moved into that playbook directly?

Reply:
Bah, the one thing I didn't test for RAID. I'll readd it

Reply:
I guess all this stuff got moved to system_setup?

Reply:
Ah yeah sorry I misread the file name, they're still there but in system setup. The goal is to ensure end users can have an easier process for deciding what, exactly, we do with their nodes while still providing a soup to nuts option for end users that aren't as concerned.